src/jquery.shorten.js

```diff
@@ -131,7 +131,7 @@
                             }
                         }
                     }
-                    c = $('<div/>').html(bag + '<span class="ellip">' + config.ellipsesText + '</span>').html();
+                    c = $('<div>').html(bag + '<span class="ellip">' + config.ellipsesText + '</span></div>').html();
```

Comment:
This looks like really weird syntax where the "</div>" is set as inside the parent `<div>`?

Reading it, I think I'd prefer:
`$('<div>' + bag + '<span class="ellip">' + config.ellipsesText + '</span></div>').html();`
though I'm not totally clear on if you're just emitting the html and now putting it anwhere why not just:
` c = '<div>' + bag + '<span class="ellip">' + config.ellipsesText + '</span></div>'`

Reply:
++ to Scott's note to contribute this back to the original plugin. Playing around with it in the console, it looks like the author was wrapping stuff in a div and then telling the div to emit html presumably so htmlPrefilter can happen, so I think the way to do what you suggest is something like: 

`$('<div></div>').html(bag + '<span class="ellip">' + config.ellipsesText + '</span>').html();`